src/collection-browser.ts

```diff
@@ -368,24 +385,48 @@ export class CollectionBrowser
     const sortField = SortFieldToMetadataField[this.selectedSort];
     if (!sortField) return;
     this.sortParam = { field: sortField, direction: this.sortDirection };
+
+    this.analyticsHandler?.sendEventNoSampling({
+      category: this.analyticsCategories.default,
+      action: this.analyticsActions.sortBy,
+      label: `${sortField} - ${this.sortDirection}`,
+    });
   }
 
   private displayModeChanged(
     e: CustomEvent<{ displayMode: CollectionDisplayMode }>
   ) {
     this.displayMode = e.detail.displayMode;
+
+    this.analyticsHandler?.sendEventNoSampling({
+      category: this.analyticsCategories.default,
```

Comment:
so with the removal of `analyticsCategories` enum, we can write `category: ${this.searchContext}` - much more descriptive 👍